Examples/js/earthquakeUSGS.js

```diff
@@ -16,13 +16,8 @@
             alias: "title",
             dataType: tableau.dataTypeEnum.string
         }, {
-            id: "lat",
-            alias: "latitude",
-            dataType: tableau.dataTypeEnum.float
-        }, {
-            id: "lon",
-            alias: "longitude",
-            dataType: tableau.dataTypeEnum.float
+            id: "location",
```

Comment:
OK. I guess the tutorial needs to be updated. The schema example, still shows the longitude and latitude.  
I can dig into this and make the changes.  I"m assuming that I can just clone the dev branch, make and test changes locally, and then make a separate PR to dev. Not sure if those changes can be added to this PR to master, or if that will need to be a separate one.